enhancements/aiops/20230120-Identify-prometheus-metrics-that-match-known-anomaly-chart-patterns.md

```diff
@@ -0,0 +1,57 @@
+# Title: 
+Identify prometheus metrics that match known anomaly chart patterns
+
+## Summary: 
+This enhancement proposal offers a new feature to match prometheus metrics to a set of known pre-defined anomaly chart patterns. This feature will make operators' life easier in the process of outage diagnosis.
+
+## Use case: 
+* Monitoring metrics in a multi-cluster setup is challenging. Operators could be responsible for hundreds of clusters and each cluster has tons of different metrics. When an outage occurs, it's difficult to identify root-cause metrics to assist outage diagnosis. Users need tools to help them reduce the scope and identify anomaly metrics faster.
+
+## Benefits: 
+* makes operators' life easier in the process of outage diagnosis.
+* improve user experience. Metric data will get condensed and categorized into insights that are much easier to digest.
+* reduce mean time to resolve outages. This method will shorten the time operators taken to find out root cause.
+* reduce economic loss.
+
+## Impact: 
+* No direct impact to existing system in terms of performance. However, if this new feature is deployed in a `proactively` in the future, then it might require some resources as it would have to continuously compute for all metrics.
```

Comment:
change "deployed in a proactively in the future" to proactively deployed in the future.

Reply:
it's a typo that should refer to `proactive mode`, added more details to it